src/content/docs/de/tutorial/0-introduction/1.mdx

```diff
@@ -0,0 +1,61 @@
+---
+type: tutorial
+title: Über dieses Tutorial
+description: Wie du mit dem "Baue deinen ersten Astro Blog" anfangen kannst.
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+
+
+## Was muss ich wissen, um loszulegen?
+
+Wenn du ein paar Grundkenntnisse mit **HTML**, **Markdown**, **CSS** und ein wenig **JavaScript** hast, kannst du direkt loslegen. 
+Du kannst das gesamte Tutorial durch gehen, in dem du den Anweisungen folgst. Astro ist für alle da! 🧑‍🚀 👩‍🚀 👨‍🚀 
+
+Außerdem brauchst du einen [GitHub](https://github.com) Account (oder etwas vergleichbares), um dein Projekt im Internet zu veröffentlichen. 
+
+<details>
+<summary>Wie kann ich die Checklisten am Ende jedes Kapitels nutzen?</summary>
+
+Du hakst sie einfach ab! 
+
+Am Ende jeder Seite befinden sich anklickbare Checklisten mit den Aufgaben, die du nun lösen können solltest. Hake diese Punkte ab, um deine Fortschritte im Tutorial Tracker verfolgen zu können. 
+Der Tracker ist nicht verpflichtend, kann dir aber dabei helfen, den Einstieg an der richtigen Stelle wieder zu finden, wenn du das Tutorial in mehreren Sitzungen abschließt. 
+Du kannst auch einige Punkte der Checkliste erst mal unabgehakt lassen, als Erinnerung, falls du später zurückkehren möchtest. 
+(Diese Daten werden nur in dem lokalen Speicher deines Browsers gespeichert und nirgendwo sonst. Keine Daten werden von Astro verschickt oder gespeichtert.)
+
+</details>
+
+<details>
+<summary>Einheit 1 behandelt Themen, die ich schon kenne. Kann ich sie überspringen?</summary>
+
+Du kannst die Lektionen von [Einheit 1](/en/tutorial/1-setup/) nutzen, um sicherzugehen, dass du die Entwicklungstools und Accounts hast, die du brauchst, um das Tutorial abzuschließen. Die Einheit begleitet dich dabei, dein erstes Astro-Projekt zu erstellen, auf GitHub zu speichern und auf Netlify bereitzustellen. 
+
+Wenn du [ein neues, leeres Astro-Projekt erstellst](/en/install/auto/) und mit dem Setup bereits vertraut bist, kannst du diese Einheit aber überspringen und direkt zu [Einheit 2](/en/tutorial/2-pages/) gehen, wo du neue Seiten für dein Projekt erstellst. 
+</details>
+
+<details>
+<summary>Was, wenn ich Hilfe brauche oder mehr über Astro erfahren will?</summary>
+
+Unser [Astro Discord Server](https://astro.build/chat) ist der beste Ort dafür. 
+
+In unserem Support Forum Channel kannst du Fragen stellen,  "Hallo" sagen oder mit uns in `#general` oder `#off-topic` chatten. 
```

Comment:
Das sollte man etwas umformulieren, vielleicht: 

"In unserem Support-Kanal kannst du Fragen stellen, oder du kannst in `#general` oder `#off-topic` "Hallo" sagen und chatten."